docs/bpftool-cgroup.rst

```diff
@@ -52,98 +52,99 @@ DESCRIPTION
+        flags or the **override** flag. Attaching another program will release
         old program and attach the new one.
 
         Multiple programs are allowed to be attached to a cgroup with **multi**.
-        They are executed in FIFO order (those that were attached first, run 
+        They are executed in FIFO order (those that were attached first, run
         first).
 
-        Non-default *ATTACH_FLAGS* are supported by kernel version 4.14 and 
+        Non-default *ATTACH_FLAGS* are supported by kernel version 4.14 and
         later.
 
         *ATTACH_TYPE* can be on of:
-        **ingress** ingress path of the inet socket (since 4.10);
 
-        **egress** egress path of the inet socket (since 4.10);
+        - **ingress** ingress path of the inet socket (since 4.10);
 
-        **sock_create** opening of an inet socket (since 4.10);
+        - **egress** egress path of the inet socket (since 4.10);
 
-        **sock_ops** various socket operations (since 4.12);
+        - **sock_create** opening of an inet socket (since 4.10);
 
-        **device** device access (since 4.15);
+        - **sock_ops** various socket operations (since 4.12);
 
-        **bind4** call to bind(2) for an inet4 socket (since 4.17);
+        - **device** device access (since 4.15);
 
-        **bind6** call to bind(2) for an inet6 socket (since 4.17);
+        - **bind4** call to bind(2) for an inet4 socket (since 4.17);
 
-        **post_bind4** return from bind(2) for an inet4 socket (since 4.17);
+        - **bind6** call to bind(2) for an inet6 socket (since 4.17);
 
-        **post_bind6** return from bind(2) for an inet6 socket (since 4.17);
+        - **post_bind4** return from bind(2) for an inet4 socket (since 4.17);
 
-        **connect4** call to connect(2) for an inet4 socket (since 4.17);
+        - **post_bind6** return from bind(2) for an inet6 socket (since 4.17);
 
-        **connect6** call to connect(2) for an inet6 socket (since 4.17);
+        - **connect4** call to connect(2) for an inet4 socket (since 4.17);
 
-        **sendmsg4** call to sendto(2), sendmsg(2), sendmmsg(2) for an 
+        - **connect6** call to connect(2) for an inet6 socket (since 4.17);
+
+        - **sendmsg4** call to sendto(2), sendmsg(2), sendmmsg(2) for an 
         unconnected udp4 socket (since 4.18);
 
-        **sendmsg6** call to sendto(2), sendmsg(2), sendmmsg(2) for an 
+        - **sendmsg6** call to sendto(2), sendmsg(2), sendmmsg(2) for an 
         unconnected udp6 socket (since 4.18);
 
-        **recvmsg4** call to recvfrom(2), recvmsg(2), recvmmsg(2) for
+        - **recvmsg4** call to recvfrom(2), recvmsg(2), recvmmsg(2) for
         an unconnected udp4 socket (since 5.2);
-        
-        **recvmsg6** call to recvfrom(2), recvmsg(2), recvmmsg(2) for
+
+        - **recvmsg6** call to recvfrom(2), recvmsg(2), recvmmsg(2) for
         an unconnected udp6 socket (since 5.2);
```

Comment:
Did you try to regenerate the docs after this change? :slightly_smiling_face: 

```
<stdin>:117: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.
<stdin>:120: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.
<stdin>:123: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.
<stdin>:126: (WARNING/2) Bullet list ends without a blank line; unexpected unindent.
```

You need to align the follow-up line on the beginning of the text (for example, on the first `*` of `**recvmesg6**`, not on the dash.

Reply:
Also, I don't mind to switching to list items. But if we do this, can we please remove the blank lines between the different items? They aren't necessary.

Reply:
Another comment on this list: would you mind fixing the typo just above it please? `*ATTACH_TYPE* can be on of:` -> `... one of:`

Reply:
No. I hadn't. Thanks for reminding me. :relieved: 
Some items extend the 80-character limit because a non-itemized line produces the following error(s).

> <stdin>:107: (ERROR/3) Unexpected indentation.
><stdin>:108: (WARNING/2) Block quote ends without a blank line; unexpected unindent.

Done. Blank lines removed.
Done. Typo corrected.

Reply:
> Some items extend the 80-character limit because a non-itemized line produces the following error(s).

This is probably because you had an issue with indent, possibly something like this:

```
        - **recvmsg4** call to recvfrom(2), recvmsg(2), recvmmsg(2) for
        an unconnected udp4 socket (since 5.2);
```

instead of this

```
        - **recvmsg4** call to recvfrom(2), recvmsg(2), recvmmsg(2) for
          an unconnected udp4 socket (since 5.2);
```

Or maybe because you're missing the empty line at the beginning of the list.

But I don't mind on one line, they're not that long and it's maybe easier to read. Let's keep like this.